File: 1_iniciante/java/1007.java
/**
 * @source Beecrowd
 * @problem 1007 - Difference
 * @subject Sequential
 * 
 * @author Letícia Cruz
 * @date 2025-07-02
 */

import java.io.IOException;
import java.util.Scanner;

class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();
        int d = scanner.nextInt();

        int diferenca = (a * b - c * d);

        System.out.println("DIFERENCA = " + diferenca);

        scanner.close();
    }
}

File: 1_iniciante/javascript/1007.js
/**
 * @source Beecrowd
 * @problem 1007 - Difference
 * @subject Sequential
 *
 * @author Letícia Cruz
 * @date 2020-10-01
 */

var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

let A = parseInt(lines.shift());
let B = parseInt(lines.shift());
let C = parseInt(lines.shift());
let D = parseInt(lines.shift());

let diferenca = A * B - C * D;

console.log("DIFERENCA = " + diferenca);
